distributed/distributed.yaml

```diff
@@ -281,10 +281,18 @@ distributed:
         - coiled
         - cudf
         - cuml
+        - pluggy  # part of pytest
         - prefect
         - rechunker
         - xarray
         - xgboost
+      ignore-files:
+        - runpy\.py  # `python -m pytest` (or other module) shell command
+        - pytest  # `pytest` shell command
+        - py\.test  # `py.test` shell command
+        - pytest-script\.py  # `pytest` shell command in Windows
+        - _pytest  # pytest implementation
+        - pycharm  # Run pytest from PyCharm GUI
```

Comment:
Could someone with vscode please run test_client.py and see if there are any regressions in the code tests?